### DISCUSSION TITLE
I have a question about importing conversation data

### DISCUSSION THREAD

**Discussion by Jasonzhu1207** [open-webui/open-webui#7846] [General]
I imported conversations.json exported from chatGPT.com into open webui. The model_slug displayed in the json file is gpt-4o, but the model displayed when the imported conversation is opened from open webui is gpt-3.5-turbo. Is this your unified processing of external data or a bug? Or does openai really swap the user's model?
Looking forward to your reply, thank you for your hard work！

**Comment by erkcan** [comment]
When we dump a chat using "Download > Export chat (.json)", we see that the model name is encoded with the "model" key in the Open WebUI's json format. Moreover the json tree structure is also different. So I think the model_slug is simply being ignored during the import.

As a crude solution, after you import the chatgpt's json file, you can export it back from Open WebUI, then replace the default "gpt-3.5-turbo" string to "gpt-4o" in a text editor (or with sed) and reimport back to Open WebUI.

`sed s/'gpt-3.5-turbo'/'gpt-4o'/g chat-export-XYZ.json > chat-to-reimport.json`
